eng/Build.props

```diff
@@ -105,8 +105,9 @@
       <ItemGroup Condition=" '$(TargetOsName)' == 'win' AND ('$(TargetArchitecture)' == 'x86' OR '$(TargetArchitecture)' == 'x64') ">
```

Comment:
What happens if someone tries to build on Windows ARM64❔

Reply:
That's kind of my main question, do we want it to continue to no-op/ignore the ancm bits, since it gets built as part of the x64 builds, so when I tried to enable it, there were missing references/restore targets.  

Maybe the simplest approach would be just to have a single windows job that built all the native stuff (win32/x86/arm64) first, and then its prebuilt for everything else, and we don't have to worry about mixing native builds

Reply:
The main issues w/ "prebuilt for everything else" are shuffling enough bits and bytes around for the other job(s) to work and the wasted time AzDO spends just starting another build job. Try w/ just one Windows x64, x86, ARM64 build job that produces all relevant native assets as well as the installers (in official builds).

This relates to a build performance improvement I intend to do early in the new year: `crossgen2` supports building for every platform from Windows x64. We could build _all_ shared Fx packages in one job and have a few other jobs that just take those packages and build installers. That's a significant change and will likely take a while&hellip;

Reply:
Ok so for this PR we'll just have one Windows build job that does everything, I think that's already working, as I see all the arm64 artifacts getting generated, they just aren't used/packaged by anything yet, I'll probably start on the installer/harvesting side when I'm back next year